com.milesalan.mepo.yml

```diff
@@ -0,0 +1,140 @@
+      - type: archive
+        url: https://github.com/libsdl-org/SDL_ttf/releases/download/release-2.20.1/SDL2_ttf-2.20.1.tar.gz
+        sha256: 78cdad51f3cc3ada6932b1bb6e914b33798ab970a1e817763f22ddbfd97d0c57
+
+  - name: SDL2_gfx
+    buildsystem: simple
+    build-commands:
+      - autoreconf -fvi
+      - ./configure --prefix=/usr --disable-mmx
+      - make install DESTDIR=/app  PREFIX=""
+    sources:
+      - type: archive
+        url: http://www.ferzkopp.net/Software/SDL2_gfx/SDL2_gfx-1.0.4.tar.gz
+        sha256: 63e0e01addedc9df2f85b93a248f06e8a04affa014a835c2ea34bfe34e576262
+
+
+  # Mepo_script runtime deps
+  - name: ncurses
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://invisible-mirror.net/archives/ncurses/current/ncurses-6.3-20220827.tgz
+        sha256: "e9daea489c50be3c325db8d49891e412a75c64aa0cfb3252d0b3d8f02529b748"
+
+  - name: dmenu
+    buildsystem: simple
+    build-commands:
+      - make install DESTDIR=/app  PREFIX="" 
+    sources:
+      - type: archive
+        url: https://dl.suckless.org/tools/dmenu-5.1.tar.gz
+        sha256: "1f4d709ebba37eb7326eba0e665e0f13be4fa24ee35c95b0d79c30f14a348fd5"
+
+  - name: bemenu
+    buildsystem: simple
+    build-commands:
+      - make install PREFIX="" DESTDIR=/app 
+    sources:
+      - type: archive
+        url: https://github.com/Cloudef/bemenu/releases/download/0.6.10/bemenu-0.6.10.tar.gz
+        sha256: "9d47557ed4572fa66e6a80364b95e4dd7a588ca75fe89c68c029b7f240b56a60"
+
+  - name: oniguruma
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://github.com/kkos/oniguruma/releases/download/v6.9.8/onig-6.9.8.tar.gz
+        sha256: "28cd62c1464623c7910565fb1ccaaa0104b2fe8b12bcd646e81f73b47535213e"
+        
+  - name: jq
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://github.com/stedolan/jq/releases/download/jq-1.6/jq-1.6.tar.gz
+        sha256: "5de8c8e29aaa3fb9cc6b47bb27299f271354ebb72514e3accadc7d38b5bbaa72"
+
+  - name: xdotool
+    buildsystem: simple
+    build-commands:
+      - make install PREFIX="" DESTDIR=/app
+    sources:
+      - type: archive
+        url: "https://github.com/jordansissel/xdotool/archive/refs/tags/v3.20211022.1.tar.gz"
+        sha256: "82b15a944a5e82fee15e0f6116bd9f642bc3d0bb6989fc0ca5ad9dfe35de0847"
+
+  # Imagemagick for icon size conversion
+  - name: imagemagick
+    sources:
+      - type: archive
+        url: https://github.com/ImageMagick/ImageMagick/archive/refs/tags/7.1.0-47.tar.gz
+        sha256: ad489af583bdfc0b5edd6c2106c955bf3af68fe838f6bd32c0ee8505ec6891ff
+
+  # Appstream metainfo
+  - name: appstream_metainfo
+    buildsystem: simple
+    builddir: true
+    build-commands:
+      - mkdir -p /app/share/metainfo/
+      - install -Dm644 com.milesalan.mepo.metainfo.xml /app/share/metainfo/com.milesalan.mepo.metainfo.xml
+    sources:
+      - type: file
+        path: com.milesalan.mepo.metainfo.xml
```

Comment:
this should with the "main" module below.

Reply:
Fixed